packages/common/lib/common/tools.js

```diff
@@ -3222,6 +3222,125 @@ function showDeprecatedMessage(func, logger) {
     logger(`Function "${func}" is deprecated. Please use async version of it: "${func}Async"`);
 }
 
+/**
+ * Requests the licenses from ioBroker.net
+ * @param {string} login Login for ioBroker.net
+ * @param {string} password Decoded password for ioBroker.net
+ * @returns {object[]} array of all licenses stored on iobroker.net
+ */
+function _readLicenses(login, password) {
+    axios = axios || require('axios');
+    const config = {
+        headers: { Authorization: `Basic ${Buffer.from(login + ':' + password).toString('base64')}` },
+        timeout: 4000
+    };
+
+    return axios.get(`https://iobroker.net:3001/api/v1/licenses`, config)
+        .then(response => {
```

Comment:
I really don't want to annoy you, but do we want to use async/await ;-)

Reply:
Rewritten